src/components/AppSettingsDialog/AppSettingsDialog.vue

```diff
@@ -298,11 +298,12 @@ export default {
 		overflow-y: auto;
 		overflow-x: hidden;
 		padding: 24px;
+		width: 100%;
 	}
 }
 
 .navigation-list {
-	height: 100%;
+	height: calc(100% - 2 * 12px);
```

Comment:
This makes sure that having many entries doesn't show an additional scrollbar on the right (if it would be then bigger than the modal size)

Reply:
done it differently now. See https://github.com/nextcloud/nextcloud-vue/pull/2197#discussion_r699447146